chapter_recurrent-neural-networks/sequence_vn.md

```diff
@@ -57,7 +72,18 @@ In fact, earthquakes are spatiotemporally correlated, i.e., the aftershocks typi
 * Humans interact with each other in a sequential nature, as can be seen in Twitter fights, dance patterns and debates.
 -->
 
-*dịch đoạn phía trên*
+* Nhiều người dùng có tần suất sử dụng ứng dụng cao ở một số thời điểm nhất định.
+Chẳng hạn, các ứng dụng truyền thông xã hội phổ biến hơn nhiều sau giờ học với học sinh.
+Các ứng dụng giao dịch trên thị trường chứng khoán được sử dụng phổ biến hơn khi thị trường mở cửa.
+* Việc dự đoán giá cổ phiếu ngày mai khó hơn nhiều so với việc dự đoán giá cổ phiếu bị bỏ lỡ ngày hôm qua, mặc dù cả hai chỉ là vấn đề ước tính một con số.
+Rốt cuộc, nhìn lại quá khứ dễ hơn nhiều so với dự đoán tương lai.
+Trong thống kê, bài toán trước được gọi là *ngoại suy* và bài toán sau được gọi là *nội suy*.
+* Âm nhạc, lời nói, văn bản, phim ảnh, bước đi, v.v ... đều có tính chất tuần tự.
+Nếu chúng ta hoán vị chúng, chúng sẽ không có ý nghĩa gì hết.
```

Comment:
```suggestion
Nếu chúng ta hoán vị chúng, chúng sẽ không còn nhiều ý nghĩa.
```
Ý gốc là không có nhiều ý nghĩa, còn em dịch phủ định hoàn toàn.

Reply:
Vâng, chỗ này khi dịch em bị nhầm. Cám ơn anh nhé!